sdk/cosmos/azure-cosmos/README.md

```diff
@@ -950,53 +950,35 @@ However, if you desire to use the CosmosHttpLoggingPolicy to obtain additional i
-    ```python
-    from azure.cosmos import CosmosClient, CosmosDiagnosticsHandler
-    import logging
-    # Initialize the logger
-    logger = logging.getLogger('azure.cosmos')
-    logger.setLevel(logging.INFO)
-    file_handler = logging.FileHandler('diagnostics1.output')
-    logger.addHandler(file_handler)
-    diagnostics_handler = cosmos_diagnostics_handler.CosmosDiagnosticsHandler()
-    diagnostics_handler["duration"] = lambda x: x > 2000
-    client = CosmosClient(URL, credential=KEY,logger=logger, diagnostics_handler=diagnostics_handler, enable_diagnostics_logging=True)
-    
-    ```
-- Using a dictionary with the relevant functions to filter out the logs you don't want to see.
-    ```python
-    # Initialize the logger
-    logger = logging.getLogger('azure.cosmos')
-    logger.setLevel(logging.INFO)
-    file_handler = logging.FileHandler('diagnostics2.output')
-    logger.addHandler(file_handler)
-    diagnostics_handler = {
-        "duration": lambda x: x > 2000
-    }
-    client = CosmosClient(URL, credential=KEY,logger=logger, diagnostics_handler=diagnostics_handler, enable_diagnostics_logging=True)
-    ```
-- Using a function that will replace the should_log function in the CosmosHttpLoggingPolicy which expects certain paramameters and returns a boolean. **Note: the parameters of the custom should_log must match the parameters of the original should_log function as shown in the sample.**
-  ```python
-  # Custom should_log method
-  def should_log(self, **kwargs):
-      return kwargs.get('duration') and kwargs['duration'] > 2000
-  
-  # Initialize the logger
+**NOTICE: The Following is a Preview Feature.**
+To further customize what gets logged, you can use logger filters to filter out the logs you don't want to see. You are able to filter based on the following attributes in the log record of cosmos diagnostics logs:
+- `status_code`
+- `duration`
+- `verb`
+- `database_name`
+- `collection_name`
+- `operation_type`
+- `url`
+- `resource_type`
+- `is_request`
+
+You can take a look at the samples [here][cosmos_diagnostics_filter_sample] or take a quick look at this snippet:
+- Using **filters** from the **logging** library, it is possible to filter the diagnostics logs. Several filterable attributes are made available to the log record of the diagnostics logs when using logging filters.
+```python
+  import logging
+  from azure.cosmos import CosmosClient
   logger = logging.getLogger('azure.cosmos')
   logger.setLevel(logging.INFO)
-  file_handler = logging.FileHandler('diagnostics3.output')
+  file_handler = logging.FileHandler('diagnostics.output')
   logger.addHandler(file_handler)
-  
-  # Initialize the Cosmos client with custom diagnostics handler
-  client = CosmosClient(endpoint, key,logger=logger, diagnostics_handler=should_log, enable_diagnostics_logging=True)
-    ```
-
+  # Create a filter to filter out logs
+  class CustomFilter(logging.Filter):
```

Comment:
I think the sample should apply a filter similar how we do it in Java default logger - filter based on status code plus duration - and for status code filter out status codes that can easily be noisy (404/0, 409/0,  412/0)